clients/users.go

```diff
@@ -135,3 +135,31 @@ func (c *UsersClient) Delete(ctx context.Context, id string) (int, error) {
 	}
 	return status, nil
 }
+
+// GetMemberGroups returns a list of Groups the user is member of, optionally filtered using OData.
+func (c *UsersClient) GetMemberGroups(ctx context.Context, id string, filter string) (*[]models.Group, int, error) {
```

Comment:
Could we rename this to `ListGroupMemberships`?

Reply:
No problem! I'm using the name that's in the graphrbac client right now, was thinking it would be easier to use the same.

I'll update tomorrow!